docs/FAQ/gui-faq-da.md

```diff
@@ -0,0 +1,151 @@
+13. [Hvordan tester jeg en "reverse" zone med Zonemaster?](#q13)
+
+Zonemaster
+----------
+#### 1. Hvad er Zonemaster? <a name="q1"></a>
+Zonemaster er et program, der er designet til at hjælpe med at tjekke, måle
+og forhåbentlig også forstå DNS (Domain Name System).
+
+Zonemaster består af tre grundlæggende moduler:
+
+  - Motor (Koden der udfører alle DNS-tests)
+  - Kommandolinjeinterface (CLI)
+  - Webinterface 
+
+Når et domænenavn (som eksempelvis "zonemaster.dk") afleveres til Zonemaster (CLI eller
+Webinterface) vil Zonemaster undersøge domænenavnets generelle sundhed ved at gennemløbe
+DNS fra roden (.) til TLD (Top Level Domæne, eksempelvis .dk) og navneservere, der indeholder
+informationer om det specifikke domænenavn ("zonemaster.dk"). De forskellige sundhedstjek
+foretaget af Zonemaster er dokumenteret i [Test Requirements document](https://github.com/dotse/zonemaster/blob/master/docs/requirements/TestRequirements.md)
+
+#### 2. Hvem står bag Zonemaster? <a name="q2"></a>
+Zonemaster er et fælles projekt mellem IIS (administrator af .se og .nu) og Afnic
+(administrator af .fr samt oversøiske territorier som tilhører Frankrig).
+
+#### 3. Hvordan kan Zonemaster hjælpe mig? <a name="q3"></a>
+Zonemaster er orienteret mod to forskellige typer mennesker:
+
+  - Mennesker som ved hvordan DNS fungerer og
+  - Mennesker som gerne vil vide, om et givent domænenavn er konfigureret korrekt.
+
+Den anden kategori kan med fordel kontakte den første kategori med spørgsmål til
+resultater af testen (konfigurationen).
+
+#### 4. Zonemaster rapporterer advarsler/fejl på mit domænenavn, hvad betyder det? <a name="q4"></a>
+Det kommer naturligvis an på hvilke advarsler/fejl, der rapporteres. I de fleste
+tilfælde kan du trykke på den aktuelle "advarsler/fejl"-besked og få detaljerede
+informationer om problemet.
+
+#### 5. Hvordan kan Zonemaster afgøre hvad der er rigtigt og forkert? <a name="q5"></a>
+Der er ikke nogen endegyldig dom over et domænenavns sundhed. Folkene bag Zonemaster påstår ikke,
+at værktøjet er korrekt i ethvert aspekt. Sommetider er meninger forskellige, især mellem lande,
+men nogle gange også lokalt. Vi har gjort vores bedste for at skabe en standard (politik)
+for fundne fejl inden dette projekt. Forhåbentlig har vi sørget for et godt kompromis mellem,
+hvad der er en egentlig potentielt farlige fejl, og hvad der blot ses som en meddelelse eller
+advarsel. Den ekstra fordel ved værktøjet er, at man kan tilføje sin egen politik og bede
+Zonemaster om at bruge denne politik til at afvikle tests.
+Men som alle andre ting udvikler DNS sig også, og det er sandsynligt, at en advarsel kan blive
+til en fejl på et senere tidspunkt. Hvis du mener, at Vi har lavet en fejl, så tøv ikke med at
+sende os en e-mail på adressen zonemaster-devel@lists.iis.se med et link til din test
+og en forklaring på, hvorfor du mener, at resultatet viser noget, som du anser forkert.
+
+####6. Understøtter Zonemaster IPv6? <a name="q6"></a>
```

Comment:
There must be a space " " between "####" and "6" to be correctly rendered as a headline. There is the same problem in the following headlines.